cfgov/unprocessed/apps/youth-employment-success/js/util.js

```diff
@@ -3,6 +3,7 @@ let UNDEFINED;
 const REDUCER_RETURN_ERROR = 'Reducer must return a state object';
 const INVALID_ARG_ERROR = 'The "reducers" argument must be an object, where each value is a reducer function';
 const INVALID_OBJECT_ERROR = 'The `entries` function must be passed an object as its first argument';
+const HTML_MINUS = '&#8722;';
```

Comment:
Nice 👍

Reply:
Yeah I figured if we have to have the entity, I might as well document what it is 😄